src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/credentials/BitbucketUsernamePasswordAuthenticator.java

```diff
@@ -64,8 +70,9 @@ public BitbucketUsernamePasswordAuthenticator(StandardUsernamePasswordCredential
     @Override
     public void configureContext(HttpClientContext context, HttpHost host) {
         CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
-        credentialsProvider.setCredentials(AuthScope.ANY, httpCredentials);
+        credentialsProvider.setCredentials(new AuthScope(host), httpCredentials);
         AuthCache authCache = new BasicAuthCache();
+		LOGGER.log(Level.SEVERE,"Add host={0} to authCache.", host);
```

Comment:
verbose logging and tabs 😓

```suggestion
        LOGGER.log(Level.FINE,"Add host={0} to authCache.", host);
```